### PR TITLE
POWR-3741 Create single display mode to display Elected Official on site menu

### PR DESCRIPTION
**Pull request by brittanysmart**

### CODE REVIEW

File: web/sites/default/config/core.entity_view_display.group.elected_official.default.yml [hunk 1]
```diff
@@ -64,14 +64,14 @@ third_party_settings:
         - field_person_image
       meta:
         - field_summary
+        - field_contact
```
Comment (joshuami) [inline web/sites/default/config/core.entity_view_display.group.elected_official.default.yml:67]: I'll leave a note in the Jira issue, but we need to lead contact in the contact region to keep the fix we did for electeds with more than one contact.

**Review by joshuami — CHANGES_REQUESTED** [review]

**Comment by joshuami** [comment]
@RichardDavies , could you give this PR a second set of eyes. I like the tile refactor work, but I don't know the cloudy-global-menu implementation quite as well as you do.

File: web/sites/default/config/core.entity_view_display.group.elected_official.default.yml (hunk 1 above)
Comment (brittanysmart) [reply web/sites/default/config/core.entity_view_display.group.elected_official.default.yml]: Good catch. Just added the fix.

**Review by RichardDavies — APPROVED** [review]
Looks good. I don't see any issues.

**Review by joshuami — APPROVED** [review]

**Pull request merged**